ecl/network/v2/load_balancer_interfaces/results.go

```diff
@@ -0,0 +1,98 @@
+package load_balancer_interfaces
+
+import (
+	"github.com/nttcom/eclcloud"
+	"github.com/nttcom/eclcloud/pagination"
+)
+
+type commonResult struct {
+	eclcloud.Result
+}
+
+// UpdateResult represents the result of an update operation. Call its Extract
+// method to interpret it as a Load Balancer Interface.
+type UpdateResult struct {
+	commonResult
+}
+
+// Extract is a function that accepts a result and extracts a Load Balancer Interface resource.
+func (r commonResult) Extract() (*LoadBalancerInterface, error) {
+	var s struct {
+		LoadBalancerInterface *LoadBalancerInterface `json:"load_balancer_interface"`
+	}
+	err := r.ExtractInto(&s)
+	return s.LoadBalancerInterface, err
+}
+
+// GetResult represents the result of a get operation. Call its Extract
+// method to interpret it as a Load Balancer Interface.
+type GetResult struct {
+	commonResult
+}
+
+// Properties used for virtual IP address
+type VirtualIPProperties struct {
+	Protocol string `json:"protocol"`
+	Vrid     int    `json:"vrid"`
+}
+
+// LoadBalancerInterface represents a Load Balancer Interface. See package documentation for a top-level
+// description of what this is.
+type LoadBalancerInterface struct {
+
+	// Description is description
+	Description string `json:"description"`
+
+	// UUID representing the Load Balancer Interface.
+	ID string `json:"id"`
+
+	// IP Address
+	IPAddress string `json:"ip_address"`
+
+	// The ID of load_balancer this load_balancer_interface belongs to.
+	LoadBalancerID string `json:"load_balancer_id"`
+
+	// Name of the Load Balancer Interface
+	Name string `json:"name"`
+
+	// UUID of the parent network.
+	NetworkID string `json:"network_id"`
```

Comment:
https://ecl.ntt.com/en/documents/api-references/network/load_balancer/load_balancer_interface.html#sample-response-body
```suggestion
	NetworkID *string `json:"network_id"`
```